apps/nextjs/src/components/AppComponents/Chat/chat-list.tsx

```diff
@@ -81,6 +86,35 @@ export function ChatList({
 
   return (
     <div className="relative flex w-full flex-col " onScroll={handleScroll}>
+      <ChatMessagesDisplay
+        messages={messages}
+        id={id}
+        lastModeration={lastModeration}
+        persistedModerations={persistedModerations}
+        ailaStreamingStatus={ailaStreamingStatus}
+      />
+
+      {isDemoLocked && <DemoLimitMessage id={id} />}
+      <div ref={chatEndRef} />
+    </div>
+  );
+}
+
+export const ChatMessagesDisplay = ({
```

Comment:
I've moved this here so that when you use the React Developer Tools plugin you can see the messages that are being rendered